src/software/world/robot.cpp

```diff
@@ -165,64 +158,57 @@ AngularVelocity Robot::estimateAngularVelocityAtFutureTime(
 
 std::vector<Point> Robot::getPreviousPositions() const
```

Comment:
Do we want to keep these `getPrevious<stuff>` functions still now that we have state? I think I would prefer only having a `getPreviousStates` function instead of individual ones for each state member variable?

Reply:
@garethellis0 @jonathanlew @scveloso thoughts on this?

Reply:
Agreed.

Reply:
+1 on only having ```getPreviousStates```

Reply:
Agreed

Reply:
refactored to ball and robot to have only getPreviousStates